README.md

```diff
@@ -5,6 +5,8 @@
 2. Install the dependencies: `npm install`.
 3. Run the tests `npm test`.
 
+What are your thoughts on using aws sam to build and deploy this stuff? I think this is totally legit, just wondering what your thought process
+behind going the bare metal/hand rolled route was?
```

Comment:
I'm using sam to upload the S3 object and deploy the lambda. I prefer to use the bare metal approach to create the actual zip file to upload b/c I feel like I have more control over the final package and because copying the files and creating the zip is a simple enough process to do by hand.